.github/workflows/e2e-nextjs.yml

```diff
@@ -2,9 +2,9 @@ name: Next.js Integration test
 
 on:
   pull_request:
-    branches: [main]
-    # Run on release PRs only
-    paths: ['CHANGELOG.md']
+    branches: [main, 'chore/fix-ci-npm-cache']
+    # Run on release PRs only, or changes to this same file
+    paths: ['CHANGELOG.md', '.github/workflows/e2e-nextjs.yml']
```

Comment:
Should we remove this change before merging?

Reply:
Actually, it kind of makes sense to run the CI when the workflow file changes, so maybe we should leave it?

Reply:
If folks agree with it I would keep both these changes ☝️ similart to what we have with `fossa` -https://github.com/netlify/zip-it-and-ship-it/blob/main/.github/workflows/fossa.yml#L6-L7 - this way we have the ability to always test this during dev if needed be?

Reply:
We could probably change the branch name though, maybe to something like `chore/ci-e2e-nextjs`? What do you think @eduardoboucas?

Reply:
Oh, I see. So you'll use that branch whenever you want to test the flow? Sounds good to me!